spec/index.bs

```diff
@@ -7349,13 +7349,29 @@ interface GPUSwapChain {
 GPUSwapChain includes GPUObjectBase;
 </script>
 
-In the "update the rendering [of the] `Document`" step of the "Update the rendering" HTML processing
-model, the contents of the {{GPUTexture}} most recently returned by
-{{GPUSwapChain/getCurrentTexture()}} are used to update the rendering for the `canvas`, and it is as
-if {{GPUTexture/destroy()}} were called on it (making it unusable elsewhere in WebGPU).
+{{GPUSwapChain}} has the following internal slots:
 
-Before this drawing buffer is presented for compositing, the implementation shall ensure that all
-rendering operations have been flushed to the drawing buffer.
+<dl dfn-type=attribute dfn-for="GPUSwapChain">
+    : <dfn>\[[format]]</dfn> of type {{GPUTextureFormat}}.
+    ::
+        The format of the textures returned by this swap chain.
+
+    : <dfn>\[[textureUsage]]</dfn> of type {{GPUTextureUsage}}.
+    ::
+        The usage of the textures returned by this swap chain.
+
+    : <dfn>\[[dirty]]</dfn> of type {{boolean}}, initially `false`.
```

Comment:
Good question. I think it's reasonable to say that newly created swap chains start off with a transparent black `currentTexture`. That would have the effect of making a `configureSwapChain()` an implicit clear operation on the canvas. There may be some minor situations in which persisting a previous swap chains frame may be useful but it's outweighed by the complications the memory management that it would introduce and if it was absolutely necessary you could get the same effect by managing the swaps explicitly with an `OffscreenCanvas`.

I've updated the text to indicate that swap chains are initialized with a texture AND that they start off being flagged as dirty.